src/PersonObjects/Grafting/ui/GraftingRoot.tsx

```diff
@@ -99,13 +99,18 @@ export const GraftingRoot = (): React.ReactElement => {
       <Typography>
         You find yourself in a secret laboratory, owned by a mysterious researcher.
         <br />
-        The scientist explains that they've been studying Augmentation grafting, the process of applying Augmentations
+        The scientist explains that they've been studying augmentation grafting, the process of applying augmentations
         without requiring a body reset.
         <br />
         <br />
-        Through legally questionable connections, the scientist has access to a vast array of Augmentation blueprints,
-        even private designs. They offer to build and graft the Augmentations to you, in exchange for both a hefty sum
+        Through legally questionable connections, the scientist has access to a vast array of augmentation blueprints,
+        even private designs. They offer to build and graft the augmentations to you, in exchange for both a hefty sum
         of money, and being a lab rat.
+        <br />
+        <br />
+        Some augmentations have prerequisites. You normally must install the prerequisites before being able to buy and
```

Comment:
Optional: Might also be worth adding to Documentation/doc/advanced/grafting.md (that file is surprisingly sparse)